staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -779,6 +779,42 @@ func (s *store) GetList(ctx context.Context, key string, opts storage.ListOption
 	}()
 
 	aggregator := s.listErrAggrFactory()
+
+	processListItem := func(kv *mvccpb.KeyValue) error {
```

Comment:
Please don't use anonymous functions.

Reply:
Changed to method, but I am not sure if it helped readability-wise. What approach do you suggest?